PocketKit/Sources/PocketKit/Home/RecommendationCarouselCell.swift

```diff
@@ -35,30 +36,21 @@ class RecommendationCarouselCell: HomeCarouselItemCell {
             viewModel.primaryAction
         }
 
+        var attributedCollection: NSAttributedString? {
+            guard viewModel.recommendation.item.isCollection else { return nil }
+            return NSAttributedString(string: Localization.Collection.title, style: .recommendation.collection)
+        }
+
         var attributedTitle: NSAttributedString {
-            return NSAttributedString(string: viewModel.title ?? "", style: .title)
+            return NSAttributedString(string: viewModel.title ?? "", style: .recommendation.title)
```

Comment:
Do collections not require titles?

Reply:
this was already pre-existing and not necessarily specific to collections itself. It seems that items are optional on the graph so not necessarily required.